app/javascript/controllers/polygon_search_controller.js

```diff
@@ -73,8 +73,9 @@ export default class extends Controller {
   }
 
   onGeojsonChange({ detail: geoJSON }) {
-    const geoJsonFeatures = geoJSON.features
+    const geoJsonFeatures = geoJSON["EPSG:27700"].features
     if (geoJsonFeatures === undefined) {
+      console.log("No geojson features were found")
```

Comment:
Do we want to be printing output to the console or is this just for debugging?

Reply:
Was left in for debugging so I'll remove